backend/module/chaos/experimentation/xds/poller.go

```diff
@@ -0,0 +1,297 @@
+package xds
```

Comment:
`xds_cache` was renamed to `Poller` + `Poller` object was introduced. It does exactly the same things as xds_cache did - it's not longer a set of 'global' methods tho